src/main.rs

```diff
@@ -60,7 +71,17 @@ fn build_subcommands<'a, 'b>(
         }
         cli_app = cli_app.subcommand(subcmd);
     }
-    cli_app
+
+    // This is needed to prevent clap from complaining. It should be removed once
+    // clap 3.x is released. See https://github.com/clap-rs/clap/issues/748
+    let custom_maskfile_path = Arg::with_name("maskfile")
+        .help("Path to a different maskfile you want to use")
+        .short("m")
+        .long("maskfile")
+        .takes_value(true)
+        .multiple(false);
+
+    cli_app.arg(maskfile)
```

Comment:
Whoops i forgot to suggest this.

```suggestion
    cli_app.arg(custom_maskfile_path)
```